.github/scripts/release_resolve_repository_context.py

```
@@ -0,0 +1,126 @@
+import os
+import urllib.error
+import urllib.parse
+import urllib.request
+from pathlib import Path
+from typing import Any
+
+OSSRH_BASE = "https://ossrh-staging-api.central.sonatype.com"
+
+
+def request_json(url: str, headers: dict[str, str]) -> tuple[int | None, dict[str, Any]]:
+    request = urllib.request.Request(url=url, method="GET", headers=headers)
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)